instructions/conda.rst

```diff
@@ -1,31 +1,42 @@
-===========================================
-Install mantid (framework-only) using conda
-===========================================
-
-Note that this is an unofficial distribution channel and not suitable for production use.
-It may not work as described and may disappear with little notice.
+========================================================
+Install mantid, mantidqt, or mantidworkbench using conda
+========================================================
 
 Supported platforms:
 
-* Linux-64: latest ubuntu/fedora/redhat (see below for distribution-specific instructions)
-* Mac OS X: experimental
+* Linux-64: available for 64-bit Linux distributions (tested for Ubuntu and CentOS/RHEL).
+* Mac OS X: available for all MacOS versions 10.10 and higher.
+* Windows: available for windows 10 and higher.
+
+You need to have a version of conda already installed, we recommend using `mambaforge <https://github.com/conda-forge/miniforge/releases>`_. As it adds support for `conda-forge <https://conda-forge.org/>`_
+channels by default, and allows users to by default, use mamba (replace `conda` with `mamba` in commands to use it), which is a faster version of conda for creating 
+`conda environments <https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html>`_.
+
+If not using mambaforge, add conda-forge to your channels::
+
+  $ conda config --add channels conda-forge 
+
+Installing latest release version of mantid::
+
+  $ conda install mantid -c mantid
```

Comment:
If we are recommending Mamba, it would be good to switch these all to:
`mamba [command]`

So that we steer users to our preferred platform too

Reply:
We'll leave these as-is, since Mamba includes the `conda` command making this guide compatible with existing Conda users